cmd/node.go

```diff
@@ -26,6 +27,42 @@ import (
 	"gopkg.in/DataDog/dd-trace-go.v1/profiler"
 )
 
+var (
+	// ErrAlreadyStarted is returned when somebody tries to start an already
+	// running service.
+	ErrAlreadyStarted = errors.New("already started")
+	// ErrAlreadyStopped is returned when somebody tries to stop an already
+	// stopped service (without resetting it).
+	ErrAlreadyStopped = errors.New("already stopped")
+	// ErrNodeNeverStarted is returned when somebody tries to find or changge the
+	// running status of the server without never starting it once.
+	ErrNodeNeverStarted = errors.New("never started the node instance")
+	// Cannot set node status to NEVERSTARTED.
+	// NEVERSTARTED is the default status, cannot set deliberately.
+	ErrCannotSetToNeverStarted = errors.New("cannot set the node status to neverstarted")
+	// Error if the atomic compare of swap of the node status fails
+	ErrFailedtoChangeNodeStatus = errors.New("failed to change the status of the node")
```

Comment:
this variable does not appear to be used - if so, can we please remove it? If we do keep it, let's fix the capitalization on the name
```suggestion
	ErrFailedToChangeNodeStatus = errors.New("failed to change the status of the node")
```